<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>report</class>
 <widget class="QDialog" name="report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WStreamLab.ico</normaloff>WStreamLab.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Common styles */
* {
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
}

/* QPushButton */
QPushButton {
    padding: 5px 10px;
    border: 2px solid #ccc; /* Light gray border */
    background-color: #f0f0f0; /* Light gray background */
    border-radius: 5px; /* Rounded corners */
}

QPushButton:hover {
    background-color: #d7d9d7; /* Light gray background on hover */
    border: 2px solid #aaa; /* Darker gray border on hover */
}

/* QComboBox */
QComboBox {
    border: 2px solid #ADD8E6; /* Light blue border */
    border-radius: 4px; /* Rounded corners */
    padding: 2px;
    min-height: 28px; /* Adjust the minimum height */
    background-color: #f0f0f0; /* Light gray background */
}

QComboBox QAbstractItemView {
    border: 2px solid #ADD8E6; /* Light blue border for the dropdown */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px; /* Adjust width as needed */
    border-left: 1px solid #ADD8E6; /* Light blue border for the drop-down button */
}

QComboBox::down-arrow {
    width: 0;
    height: 0;
}

QComboBox::down-arrow:disabled {
    image: url(circle.png); /* Replace with your own circle image */
    width: 16px; /* Adjust width of circle */
    height: 16px; /* Adjust height of circle */
}

/* QLineEdit */
QLineEdit {
    border: 2px solid #ADD8E6; /* Light blue border */
    border-radius: 4px; /* Rounded corners */
    height: 28px; /* Adjusted height */
    padding: 2px; /* Adjust padding if needed */
    background-color: #f0f0f0; /* Light gray background */
}

/* QRadioButton */
QRadioButton::indicator {
    width: 14px; /* Adjusted width */
    height: 14px; /* Adjusted height */
    border-radius: 7px;
}

QRadioButton::indicator:checked {
    background-color: black;
    border: 4px solid black; /* Added border style */
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 4px solid black; /* Added border style */
}

/* QGroupBox */
QGroupBox {
    border: 2px solid black;
    border-color: rgba(0, 0, 0, 0.36); /* Black with transparency */
    background-color: #E6E6FF; /* Light purple background */
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    margin-top: 10px;
    border-radius: 8px; /* Rounded corners */
    padding-top: 10px; /* Padding for the top of the group box */
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 5px; /* Adjusted padding for better appearance */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grBoxBuletin">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="lbCodulDinLt">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lbCost">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leAutorizatiaNumarul">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leBeneficiar">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbBeneficiar">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbAutorizatiaNumarul">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="lbLoculEfectuariiVerificarii">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leCoduldinLt">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cbValabilitate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leNumarInregistrare">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="leCost">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbNumarInregistrare">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leNormativ">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lbVerificatorMetrolog">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lbValabilitate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="leVerificatorMetrolog">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbNormativ">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="leLoculEfectuariiVerificarii">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbGenerareBV">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbInchide">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leNumarInregistrare</tabstop>
  <tabstop>leBeneficiar</tabstop>
  <tabstop>leCoduldinLt</tabstop>
  <tabstop>leNormativ</tabstop>
  <tabstop>cbValabilitate</tabstop>
  <tabstop>leCost</tabstop>
  <tabstop>leVerificatorMetrolog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
